src/routes/(app)/events/InterestedGoingList.svelte

```diff
@@ -2,33 +2,60 @@
   import { getFullName } from "$lib/utils/client/member";
   import type { Member } from "@prisma/client";
   import AuthorSignature from "$lib/components/AuthorSignature.svelte";
+  import * as m from "$paraglide/messages";
 
   let goingModal: HTMLDialogElement;
   let interestedModal: HTMLDialogElement;
   export let interested: Member[];
   export let going: Member[];
 
   $: goingText =
-    going.length > 2
-      ? `${getFullName(going[0]!)}, ${getFullName(going[1]!)} och ${
-          going.length - 2
-        } andra`
-      : `${going.map((m) => getFullName(m)).join(" och ")}`;
+    going.length > 0
+      ? going.length === 1
+        ? m.events_interestedGoing_isGoing({ x: getFullName(going[0]!) })
+        : m.events_interestedGoing_areGoing({
+            x:
+              going.length > 2
+                ? m.events_interestedGoing_threeOrMore({
+                    name1: getFullName(going[0]!),
+                    name2: getFullName(going[1]!),
+                    others: going.length - 2,
+                  })
+                : m.events_interestedGoing_two({
+                    name1: getFullName(going[0]!),
+                    name2: getFullName(going[1]!),
+                  }),
+          })
+      : "";
```

Comment:
this feels a bit cluttered. I don't know for paraglide but for other i18n libs there is usually native support for pluralisation. I'm not sure if that works here, but nonetheless this should be moved to perhaps a function or something with clearer flow. It took me a few seconds to understand what happens here, and if we just split it up to a few lines of codes with some if-elses I think if will be much clearer.